tests/defined_resource_types_spec.rb

```diff
@@ -71,7 +110,12 @@
   end
 end
 describe "Task 6:", host: :pastureappsmall do
-  it 'Trigger a Puppet run on pasture-app-small.puppet.vm to enforce your changes' do
+  it 'has a working solution', :solution do
+    command("puppet job run --nodes pasture-app-small.puppet.vm")
```

Comment:
@greglarkin I believe you can set the host per `it` block just like you can for a `describe` block. I'll try it out.